packages/validator-ajv8/test/validator.test.ts

```diff
@@ -32,10 +25,10 @@ describe('AJV8Validator', () => {
     builder.resetAllErrors();
   });
   describe('default options', () => {
-    // Use the TestValidator to access the `withIdRefPrefix` function
-    let validator: TestValidator;
+    // Use the AJV8Validator to access the `withIdRefPrefix` function
```

Comment:
Does it make sense to remove this comment (and the other 3 times it appears in this file)? Seems like the refactor made this easy to clean up and we're not even accessing `withIdRefPrefix` in this file
```suggestion
```

Reply:
Good catch!